tests/controls/textctrltest.cpp

```diff
@@ -275,51 +276,134 @@ void TextCtrlTestCase::ReadOnly()
 void TextCtrlTestCase::MaxLength()
 {
 #if wxUSE_UIACTIONSIMULATOR
+    wxUIActionSimulator sim;
+
+    if ( ms_style == wxTE_MULTILINE )
+    {
+#if defined(__WXMSW__) /*|| defined(__WXGTK3__)*/ || defined(__WXQT__)
```

Comment:
@oneeyeman1 Your PR works correctly with these tests. Just uncomment this line.

Reply:
@AliKet ,
Thx for checking. All there is to do is fix the comments. Will do that sometime next week.

Do you have access to a Mac? If not - I'll try that later...

Reply:
> Do you have access to a Mac? If not - I'll try that later...

Unfortunately not, sorry.